paperspace/commands/jobs.py

```diff
@@ -4,9 +4,10 @@
 from click import style
 
 from paperspace import config, client
-from paperspace.commands import CommandBase
+from paperspace.commands.common import CommandBase
 from paperspace.utils import get_terminal_lines
 from paperspace.workspace import S3WorkspaceHandler
+from . import common
```

Comment:
same as above. also, why one time it's relative, other time it's absolute?

Reply:
fixed